.circleci/config.yml

```diff
@@ -34,8 +35,25 @@ jobs:
             . venv/bin/activate
             pip install -r requirements.txt
             pip install -r requirements-dev.txt
+            pip install wheel
+            # Package up the package
+            python setup.py sdist bdist_wheel
 
       - save_cache:
           paths:
             - ./venv
-          key: v1-dependencies-{{ checksum "requirements.txt" }}-{{ checksum "requirements-dev.txt" }}
+          key: v2-dependencies-{{ checksum "requirements.txt" }}-{{ checksum "requirements-dev.txt" }}
+
+      - run:
+          name: integration tests
+          command: |
+            python3 -m venv venv2
```

Comment:
This is probably a bit nit picky. Can we change `venv2` to `venv_tests`? I keep thinking of Python 2 when I see the name and want to make sure others aren't confused.

Reply:
This is a great suggestion, I've even confused myself.